### ISSUE TITLE
multiple-value uuid.NewV4 Makefile error

### ISSUE THREAD

**Issue by garrettjj** [sachaos/todoist#61]
When attempting to install on my Fedora 29 install, I'm receiving the following error:

```
➜  todoist (master) ✔ cd $GOPATH/src/github.com/sachaos/todoist
➜  todoist (master) ✔ make install
go get golang.org/x/tools/cmd/goyacc
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
        ignoring go.mod;
        see 'go help modules'
goyacc -o filter_parser.go filter_parser.y

conflicts: 4 shift/reduce
go install
# github.com/sachaos/todoist/lib
lib/command.go:21:21: multiple-value uuid.NewV4() in single-value context
lib/command.go:22:21: multiple-value uuid.NewV4() in single-value context
make: *** [Makefile:3: install] Error 2
```

I've taken note from issue #36 and ensured that I'm building in the right folder although I'm sure this is likely a dumb mistake on my part since I'm running this tool perfectly on another Fedora 29 install and a Debian WSL instance.

Thanks for the great tool and thanks for any time.

**Comment by sachaos** [comment]
@garrettjj Now, this project should be build with Go 1.11 or above.
What is your Go version?

**Comment by garrettjj** [comment]
Apologies for not including that originally. It appears that I'm running 1.11.2.

➜  ~ go version
go version go1.11.2 linux/amd64

Oddly enough, this works smoothly on two other systems. One of them is even running the same flavor of Linux (Fedora) and the same dotfiles.

**Comment by hokorobi** [comment]
I think this is the same cause.
https://github.com/hlandau/acme/issues/293

**Comment by hokorobi** [comment]
Oh, maybe set `GO111MODULE=on`.

**Comment by sachaos** [comment]
This issue would be fixed by #65
